live-build/hooks/30-fix-timedatectl.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+mv /usr/bin/timedatectl /usr/bin/timedatectl.real
+
+cat >/usr/bin/timedatectl<<EOF
```

Comment:
yes, i tested it many times... copied the original timedatectl to $HOME and bind mounted the script on top of /usr/bin/timedatectl after changing the value of TIMEDATECTL to point there